cli/tests/test_completion.rs

```diff
@@ -227,22 +326,39 @@ fn test_aliases_are_resolved() {
         .run_jj(["config", "set", "--repo", "aliases.b2", "['bookmark']"])
         .success();
 
-    let mut test_env = test_env;
-    test_env.add_env_var("COMPLETE", "fish");
-    let test_env = test_env;
-    let work_dir = test_env.work_dir("repo");
-
-    let output = work_dir.run_jj(["--", "jj", "b", "rename", "a"]);
-    insta::assert_snapshot!(output, @r"
-    aaa	(no description set)
-    [EOF]
-    ");
+    let output = work_dir.complete_at(shell, 3, ["b", "rename", "a"]);
+    match shell {
+        Shell::Bash => {
+            insta::assert_snapshot!(output, @"aaa[EOF]");
+        }
+        Shell::Zsh => {
+            insta::assert_snapshot!(output, @"aaa:(no description set)[EOF]");
+        }
+        Shell::Fish => {
+            insta::assert_snapshot!(output, @r"
+                    aaa	(no description set)
+                    [EOF]
+                    ");
```

Comment:
nit: rerun `insta test --force-update-snapshots` or something to fix weird indent

Reply:
Done. Thanks!